### ISSUE TITLE
Quoting either min or max inhibits simplification

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3640]
Imported from SourceForge on 2024-07-08 10:31:15
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2021-03-10 12:36:27
Original: https://sourceforge.net/p/maxima/bugs/3736

---

Quoting either min or max inhibits simplification--this is wrong:
~~~
(%i1)	'max(1,1);
(%o1)	max(1,1)

(%i2)	'min(1,1);
(%o2)	min(1,1)
~~~

A putative fix is insert

~~~
(defprop $max $max verb)
(defprop $min $min verb)
~~~

into maxmin.lisp.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:31:17
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-03-16 05:23:28
Original: https://sourceforge.net/p/maxima/bugs/3736/#64be

---

- **labels**: max min --> max, min, quoting
- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:31:21
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-03-16 05:23:29
Original: https://sourceforge.net/p/maxima/bugs/3736/#a0cb

---

Fixed by 37b10cdb by Barton W.
